FlowCrypt/Controllers/Setup/PassPhraseStorageService.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  PassPhraseStorageService.swift
+//  FlowCrypt
+//
+//  Created by Anton Kharchevskyi on 02.06.2021.
+//  Copyright © 2021 FlowCrypt Limited. All rights reserved.
+//
+
+import UIKit
+
+final class PassPhraseStorageService {
+    struct Context {
+        let passPhrase: String
+        let keys: [KeyDetails]
+        let source: KeySource
+        let isLocally: Bool
+    }
+
+    let storage: KeyStorageType
+
+    init(storage: KeyStorageType = EncryptedStorage()) {
+        self.storage = storage
+    }
+
+    func savePassPhrase(with context: Context) {
+        if context.isLocally {
+            storage.addKeys(keyDetails: context.keys, passPhrase: context.passPhrase, source: context.source)
+        } else {
+            // TODO: - ANTON
+        }
+    }
```

Comment:
what does the `isLocally` represent here?

Reply:
I think this whole file can be deleted, I thought to make `PassPhraseStorageService` but then used just `PassPhraseStorage`. Thanks for noticing

Reply:
Ok - please remove the unused parts (don't change git history or force push) and then I can just review the last commit separately.

Reply:
sure, I forced pushed just once) Will do today

Reply:
It was just a file in setup folder. Was not used. 
Xcode replaced reference from the project, but file was not deleted. Fixed.